src/main/webapp/js/instructorFeedbackAjaxRemindModal.js

```diff
@@ -3,6 +3,7 @@ $(document).ready(function() {
         var button = $(event.relatedTarget); // Button that triggered the modal
         var actionlink = button.data('actionlink');
         
+        $('#remindBtn').prop('disabled', true).prop('value', 'Loading...');
```

Comment:
Rather than adding a new id to the button, why not select it via jQuery?
The modal has an id, so just find the submit button under it.
(It should be alright to assume there will only be one submit button for the form / modal.)